src/test/java/com/hydratereminder/chat/ChatMessageSenderTest.java

```diff
@@ -28,47 +28,46 @@ class ChatMessageSenderTest {
     @InjectMocks
     private transient ChatMessageSender chatMessageSender;
 
+    private static String  message = "Hello";
+
```

Comment:
@jmakhack I can't really understand why it's asking for a field comment, shouldn't be a good practice of clean code the use of self explanatory names for variables instead of comments?

Reply:
@ZarakiKanzaki don't worry about the `Field comments are required` comment. It can be ignored.